src/main/java/seedu/address/logic/commands/CopyMemberCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Member;
+
+/**
+ * Copies a member from the address book to the clipboard.
+ */
+public class CopyMemberCommand extends Command {
+
+    public static final String COMMAND_WORD = "copyMember";
+    public static final String COMMAND_ALIAS = "cpm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
same for this as mentioned earlier in cpa. 

Perhaps can change to : Copies the details (name, phone number, email, etc) of the applicant at the specified index to clipboard.

Reply:
same as above